config-templates/ibm/ibm-vpc-block-csi-driver/5.0/deployment.yaml

```diff
@@ -0,0 +1,734 @@
+        resources: ["csinodeinfos"]
+        verbs: ["get", "list", "watch"]
+      - apiGroups: ["storage.k8s.io"]
+        resources: ["volumeattachments"]
+        verbs: ["get", "list", "watch", "update", "patch"]
+      - apiGroups: ["storage.k8s.io"]
+        resources: ["volumeattachments/status"]
+        verbs: ["patch"]
+  - kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: vpc-block-external-attacher-binding
+      labels:
+        app: ibm-vpc-block-csi-driver
+    subjects:
+      - kind: ServiceAccount
+        name: ibm-vpc-block-controller-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: vpc-block-external-attacher-role
+      apiGroup: rbac.authorization.k8s.io
+  - kind: ClusterRole
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: vpc-block-driver-snapshotter-role
+      labels:
+        app: ibm-vpc-block-csi-driver
+        addonmanager.kubernetes.io/mode: Reconcile
+    rules:
+      - apiGroups: [""]
+        resources: ["events"]
+        verbs: ["list", "watch", "create", "update", "patch"]
+      - apiGroups: ["snapshot.storage.k8s.io"]
+        resources: ["volumesnapshotclasses"]
+        verbs: ["get", "list", "watch"]
+      - apiGroups: [""]
+        resources: ["secrets"]
+        verbs: ["get", "list"]
+      - apiGroups: ["snapshot.storage.k8s.io"]
+        resources: ["volumesnapshotcontents"]
+        verbs: ["create", "get", "list", "watch", "update", "delete", "patch"]
+      - apiGroups: ["snapshot.storage.k8s.io"]
+        resources: ["volumesnapshotcontents/status"]
+        verbs: ["update"]
+  - kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: vpc-block-driver-snapshotter-binding
+      labels:
+        app: ibm-vpc-block-csi-driver
+        addonmanager.kubernetes.io/mode: Reconcile
+    subjects:
+      - kind: ServiceAccount
+        name: ibm-vpc-block-controller-sa
+        namespace: kube-system
+    roleRef:
+      kind: ClusterRole
+      name: vpc-block-driver-snapshotter-role
+      apiGroup: rbac.authorization.k8s.io   
+  - kind: CSIDriver
+    apiVersion: storage.k8s.io/v1
+    metadata:
+      name: vpc.block.csi.ibm.io
+    spec:
+      attachRequired: true
+      podInfoOnMount: true
+      volumeLifecycleModes:
+        - Persistent
+  - apiVersion: v1
+    stringData:
+      slclient.toml: |
+          [server]
+          debug_trace = false
+          [vpc]
+          iam_client_id = "bx"
+          iam_client_secret = "bx"
+          g2_token_exchange_endpoint_url = "{{{g2_token_exchange_endpoint_url}}}"
+          g2_riaas_endpoint_url = "{{{g2_riaas_endpoint_url}}}"
+          g2_resource_group_id = "{{{g2_resource_group_id}}}"
+          g2_api_key = "{{{g2_api_key}}}"
+          provider_type = "g2"
+    kind: Secret
+    metadata:
+      name: storage-secret-store
+      namespace: kube-system
```

Comment:
Add label for this resource as well

Reply:
storage-secret-store is a resource used by other components too, Adding any label here wouldn't be correct is my opinion

Reply:
ok